README.md

```diff
@@ -80,15 +80,98 @@ See below for more details.
 
 ## Test POCS
 
-Once you have setup your computer (above), the next step is to test your setup.
-This is easy to do using our built-in test suite. In a terminal, simply type:
+POCS comes with a testing suite that allows it to test that all of the software works and is installed corretly. Running the test suite by default will use simulators for all of the hardware and is meant to test that the software works correctly. Additionally, the testing suite can be run with various flags to test that attached hardware is working properly.
+
+All of the test files live in `$POCS/pocs/tests`.
+
+### Software Testing
+
+There are two scenarios where you want to run the test suite:
+
+1. You are getting your unit ready and want to test software is installed correctly
+2. You are helping develop code for POCS and want test your code doesn't break something
```

Comment:
Please end sentences with periods (generally, please use complete sentences and normal punctuation).